internal/definition/validate.go

```diff
@@ -55,44 +55,73 @@ func ValidateDefinition(ctx context.Context, fpath string, sources []source.Poli
 }
 
 // detect if a file or directory was passed. if a directory, gather all files in it
-func detectFiles(ctx context.Context, fpath string) ([]string, error) {
+// the order is file lookup, json lookup then yaml
+func detectInput(ctx context.Context, fpath string) ([]string, error) {
+	log.Error("trying to detect input as JSON")
```

Comment:
Not sure about logging at this level, perhaps the only error we should print is when we can't detect/load from a path?

Reply:
I wasn't in love with it either. Was just trying to find a way to communicate to the user what was going on. I moved the logging to after the validation attempt and to debug. wdyt?

```golang
if utils.IsJson(fpath) {
    log.Debug("valid JSON found for definition file")
    return definitionFromString(ctx, fpath)
}
log.Debug("unable to detect input as JSON")
```